Resources/Prototypes/_LostParadise/Entities/Markers/Spawners/Random/dungeon_items_general.yml

```
@@ -0,0 +1,992 @@
+    - VendingMachineShamblersJuice
+    - VendingMachinePwrGame
+    - VendingMachineDrGibb
+    - VendingMachineSovietSoda
+    - VendingMachineGames
+    - VendingMachineHappyHonk
+    - VendingMachineCuddlyCritterVend
+    - VendingMachineAutoTuneVend
+    - VendingMachinePottedPlantVend
+    - WaterCooler
+    - SpawnDungeonMachineFrame
+    #- VendingMachineChang # No food vendomats in expeds - buy from food ships or die
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)